src/main/java/io/stargate/sgv2/jsonapi/service/embedding/operation/AwsBedrockEmbeddingProvider.java

```diff
@@ -161,6 +162,7 @@ public Uni<Response> vectorize(
   private record EmbeddingRequest(
       String inputText, @JsonInclude(value = JsonInclude.Include.NON_DEFAULT) int dimensions) {}
 
+  @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Doing this will ignore all properties, which may be ok. Alternatively could use

   @JsonIgnoreProperties({ "embeddingsByType" })

to just ignore specific unknown property. I'll let you decide which makes more sense.